src/infra/dev/eu-south-1/main.tf

```diff
@@ -36,16 +36,19 @@ module "frontend" {
   source = "../../modules/frontend"
 
   # DNS 
-  domain_name     = module.r53_zones.dns_zone_name
-  r53_dns_zone_id = module.r53_zones.dns_zone_id
-  role_prefix     = local.project
+  domain_name       = module.r53_zones.dns_zone_name
+  domain_admin_name = module.r53_zones.dns_zone_name
+  r53_dns_zone_id   = module.r53_zones.dns_zone_id
+  role_prefix       = local.project
 
   ## API Gateway ##
   rest_api_name         = format("%s-restapi", local.project)
   openapi_template_file = "../../api/oi.tpl.json"
+  rest_api_admin_name   = format("%s-restapi-admin", local.project)
 
-  dns_record_ttl     = var.dns_record_ttl
-  cors_allow_origins = "*"
+  openapi_admin_template_file = "../../api/oi-admin.tpl.json"
+  dns_record_ttl              = var.dns_record_ttl
+  cors_allow_origins          = "*"
```

Comment:
Is this required. I mean in the admin part if I'm not wrong we not need the cors configuration.